testing-utils/src/index.ts

```diff
@@ -1,6 +1,9 @@
 export * from './lib/constants';
 export * from './lib/setup/test-folder.setup';
 export * from './lib/utils/execute-process-helper.mock';
+// utils
```

Comment:
Exports from lines 1-3 are also generic utils. Not sure if this comment is necessary.

Reply:
removed